kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -24,10 +24,17 @@ const publicMethods = [
  * is configured in webpack.config.js
  * @classdesc
  */
-function Lib() {
+function Lib(kolibriGlobal) {
   // libraries
   this.loglevel = require('loglevel');
   this.vue = require('vue');
+  this.vue.mixin({
+    methods: {
+      url: (url_name, ...args) => {
+        return kolibriGlobal.urls[url_name](...args);
+      }
```

Comment:
I agree there is much less of a use case for it being used in view rendering than there was in KA Lite - and can remove the Vue mixin from this PR, until there is a strong use case for it.